crates/biome_cli/src/commands/mod.rs

```diff
@@ -277,6 +278,40 @@ pub enum BiomeCommand {
         sub_command: Option<MigrateSubCommand>,
     },
 
+    /// Searches for Grit patterns across a project.
+    #[bpaf(command, hide)] // !! Command is hidden until ready for release.
+    Search {
+        /// The GritQL pattern to search for.
+        ///
+        /// Note that the search command (currently) does not support rewrites.
+        #[bpaf(positional("PATH"))]
```

Comment:
All positional items must go at the end of the block, after all non-positional items.

Reply:
Thanks for catching! Seems `check_invariants()` didn't actually catch this, but it's a good thing to have anyway :+1:

Reply:
Hmm... That's interesting, seems like a bug. Do you have a commit with it present and not catching?